Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -715,8 +719,13 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], mirror_ta
             old_incident = client.get_incident(incident_id=parsed_args.remote_incident_id)
             for changed_key in parsed_args.delta.keys():
                 old_incident[changed_key] = parsed_args.delta[changed_key]  # type: ignore
-                if changed_key in old_incident.get('CustomFields', {}).keys():
+                if changed_key in old_incident.get('CustomFields', {}).keys() or changed_key in custom_fields:
+                    demisto.debug(f'Field {changed_key} is synced as a custom field.')  # todo remove before merge
+
                     old_incident['CustomFields'][changed_key] = parsed_args.delta[changed_key]
+                else:
+                    demisto.debug(f'Field {changed_key} is not synced as a custom field, because it is not in the remote XSOAR '
+                                  f'CustomFields nor in the integration custom_fields parameter.')  # todo remove before merge
```

Comment:
todo

Reply:
It is here in case someone else will have to view the logs or to merge this 🙂